fury/molecular.py

```diff
@@ -0,0 +1,698 @@
+    vtkSimpleBondPerceiver performs a simple check of all interatomic
+    distances and adds a single bond between atoms that are reasonably
+    close. If the interatomic distance is less than the sum of the two
+    atom's covalent radii plus a tolerance, a single bond is added.
+
+    Parameters
+    ----------
+    molecule : Molecule() object
+        The molecule for which bonding information is to be generated.
+
+    Notes
+    -----
+    This algorithm does not consider valences, hybridization, aromaticity,
+    or anything other than atomic separations. It will not produce anything
+    other than single bonds.
+    """
+    bonder = vtk.vtkSimpleBondPerceiver()
+    bonder.SetInputData(molecule)
+    bonder.SetTolerance(0.1)
+    bonder.Update()
+    deep_copy(molecule, bonder.GetOutput())
+
+
+class PeriodicTable(vtk.vtkPeriodicTable):
+    """ A class to obtain properties of elements (eg: Covalent Radius,
+    Van Der Waals Radius, Symbol etc.).
+
+    This is a more pythonic version of ``vtkPeriodicTable`` providing simple
+    methods to access atomic properties. It provides access to essential
+    functionality available in ``vtkPeriodicTable``. An object of this class
+    provides access to atomic information sourced from Blue Obelisk Data
+    Repository.
+    """
+
+    def atomic_symbol(self, atomic_number):
+        """Given an atomic number, returns the symbol associated with the
+        element.
+
+        Parameters
+        ----------
+        atomic_number : int
+            Atomic number of the element whose symbol is to be obtained.
+        """
+        return self.GetSymbol(atomic_number)
+
+    def element_name(self, atomic_number):
+        """Given an atomic number, returns the name of the element.
+
+        Parameters
+        ----------
+        atomic_number : int
+            Atomic number of the element whose name is to be obtained.
+        """
+        return self.GetElementName(atomic_number)
+
+    def atomic_number(self, element_name):
+        """Given a case-insensitive string that contains the symbol or name of
+        an element, return the corresponding atomic number.
+
+        Parameters
+        ----------
+        element_name : string
+            Name of the element whose atomic number is to be obtained.
+        """
+        return self.GetAtomicNumber(element_name)
+
+    def atomic_radius(self, atomic_number, radius_type):
+        """Given an atomic number, return either the covalent radius of the
+        atom or return the Van Der Waals radius of the atom depending on
+        radius_type.
+
+        Parameters
+        ----------
+        atomic_number : int
+            Atomic number of the element whose covalent radius is to be
+            obtained.
+        radius_type : string
+            Two valid choices -
+            * 'VDW' : for Van Der Waals radius of the atom
+            * 'Covalent' : for covalent radius of the atom
+        """
+        if radius_type == 'VDW':
```

Comment:
Do you know which one ('VDW' or 'Covalent') is by default? 
You can raise an error if radius_type is something else.

Reply:
Post comment, I made it such that it's 'VDW' by default and a value error is raised if it's something other than these two. Thanks!